bundles/org.openhab.binding.guntamatic/src/main/java/org/openhab/binding/guntamatic/internal/GuntamaticHandlerFactory.java

```diff
@@ -61,8 +73,14 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     protected @Nullable ThingHandler createHandler(Thing thing) {
         ThingTypeUID thingTypeUID = thing.getThingTypeUID();
 
+        if ((thingTypeUID == THING_TYPE_BIOSTAR) || (thingTypeUID == THING_TYPE_POWERCHIP)
+                || (thingTypeUID == THING_TYPE_POWERCORN) || (thingTypeUID == THING_TYPE_BIOCOM)
+                || (thingTypeUID == THING_TYPE_PRO) || (thingTypeUID == THING_TYPE_THERM)) {
+            staticChannelIDs.add(CHANNEL_CONTROLBOILERAPPROVAL);
```

Comment:
This is not good as you will generate duplicated entries in your list each time a new handler is created.